arch/arm/aarch64.cmake

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set_property(   GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf64-littleaarch64) # BFD format
```

Comment:
What's with the extra leading spaces here?

Reply:
Because that was copy and pasted from https://github.com/zephyrproject-rtos/zephyr/blob/master/CMakeLists.txt#L60

I'll remove the spaces.